pandas/tests/io/json/test_normalize.py

```diff
@@ -460,3 +496,59 @@ def test_nonetype_multiple_levels(self):
             'location.country.state.town.info.y': -33.148521423339844,
             'location.country.state.town.info.z': 27.572303771972656}
         assert result == expected
+
+    @pytest.mark.parametrize("max_level, input_data, expected", [
```

Comment:
so use the max_level_test_input_data here; If you have another example then make that a separate test function

Reply:
@WillAyd asked me to merge "None, 0, 1 and 100" into a single test. Should I split it into two?

Reply:
Wasn't thinking about this but this is a good request. Sorry for back and forth but if you split 100 off you can use that fixture for all of the other inputs here right? Would be more readable to go that way

Reply:
Okay will do that.